### PR TITLE
Minor/boring change: Consistently return user facing errors in SDN

### PR DESCRIPTION
**Pull request by pravisankar**
Follow the existing convention: user facing errors should always start with lowercase.

### CODE REVIEW

**Comment by pravisankar** [comment]
@openshift/networking ptal

**Comment by knobunc** [comment]
Is the lower-case convention true for acronyms too?

**Comment by pravisankar** [comment]
Hm, not sure. @smarterclayton ^^

**Comment by smarterclayton** [comment]
It would not be true for acronyms in the beginning of error messages - they can stay as is.

**Comment by pravisankar** [comment]
Updated, ptal

**Comment by pravisankar** [comment]
[test]

File: pkg/sdn/plugin/master.go [hunk 1]
```diff
@@ -120,32 +120,32 @@ func (master *OsdnMaster) validateNetworkConfig() error {
 	// Ensure cluster and service network don't overlap with host networks
 	for _, ipNet := range hostIPNets {
 		if ipNet.Contains(ni.ClusterNetwork.IP) {
-			errList = append(errList, fmt.Errorf("Error: Cluster IP: %s conflicts with host network: %s", ni.ClusterNetwork.IP.String(), ipNet.String()))
+			errList = append(errList, fmt.Errorf("error: Cluster IP: %s conflicts with host network: %s", ni.ClusterNetwork.IP.String(), ipNet.String()))
```
Comment (danwinship) [inline pkg/sdn/plugin/master.go]: Can you drop the "error:" prefix here? (4 cases in this loop, plus another 2 later on in this file)

Other than that LGTM

**Comment by pravisankar** [comment]
[merge]

**Comment by knobunc** [comment]
[merge]

**Pull request merged**